### ISSUE TITLE
Column numbering?

### ISSUE THREAD

**Issue by bmralph** [wadetb/Sublime-Text-Advanced-CSV#29]
Sure would be totally awesome if column number could be output.

I have large CSV files with many columns, I have to manually scroll through counting to work out what number column my desired data is in.

**Comment by wadetb** [comment]
Good idea! I wonder if there is an API in ST3 to output it to the status bar.

Similarly I was thinking it would be useful to have a way to copy the current row & column number to the clipboard, in the format compatible with my expression language.

**Comment by bmralph** [comment]
Sounds good. Happy you feel this feature would be helpful also.
